mockery.yaml

```diff
@@ -0,0 +1,39 @@
+dir: "{{ .InterfaceDir }}/mocks"
+mockname: "{{ .InterfaceName }}"
+outpkg: mocks
+filename: "{{ .InterfaceName | snakecase }}.go"
+packages:
+  github.com/smartcontractkit/chainlink-common/pkg/capabilities/datastreams:
+    interfaces:
+      ReportCodec:
+  github.com/smartcontractkit/chainlink-common/pkg/loop/internal/types:
+    interfaces:
+      Relayer:
+        config:
+          dir: "{{ .InterfaceDir }}/../../mocks"
+  github.com/smartcontractkit/chainlink-common/pkg/monitoring:
+    config:
+      dir: "{{ .InterfaceDir }}"
+      filename: "{{ .InterfaceName | snakecase }}_mock.go"
+      inpackage: true
```

Comment:
do you know why monitoring is different than other packages?

Reply:
I thought there used to be a whole separate module under `ops` that used it but now I'm not so sure :shrug: